docs/fio.md

```diff
@@ -0,0 +1,263 @@
+
+### ENABLE_PBENCH_AGENTS
+
+Default: `false`  
+Enables/disables the collection of pbench data on the pbench agent Pods. These Pods are deployed by the tooling playbook
+and if this option is enabled then tooling playbook needs to be executed prior to this test.
+
+### ENABLE_PBENCH_COPY
+
+Default: `true`  
+
+Enables/disables the copying of pbench data to a remote results server for further analysis.
+As of now, this test requires valid `PBENCH_SERVER` where it will copy results at end of test. 
+
+### PBENCH_SSH_PRIVATE_KEY_FILE
+
+Default: `~/.ssh/id_rsa`  
+Location of ssh private key to authenticate to the pbench results server.
+
+### PBENCH_SSH_PUBLIC_KEY_FILE
+
+Default: `~/.ssh/id_rsa.pub`  
+Location of the ssh public key to authenticate to the pbench results server.
+
+### PBENCH_SERVER
+
+Default: There is no public default.  
+DNS address of the pbench results server.
+
+### SCALE_CI_RESULTS_TOKEN
+
+Default: There is no public default.  
+Future use for pbench and prometheus scraper to place results into git repo that holds results data.
+
+### JOB_COMPLETION_POLL_ATTEMPTS
+
+Default: `10000`  
+
+Number of retries for Ansible to poll if the workload job has completed. 
+Poll attempts delay 10s between polls with some additional time taken for each polling action depending on the orchestration host setup.
+FIO I/O test for many pods and big file sizes can run for hours and either we rise `JOB_COMPLETION_POLL_ATTEMPTS` to
+higt value, or remove fully checking for `JOB_COMPLETION_POLL_ATTEMPTS` for FIO I/O test. 
+
+
+### FIOTEST_PREFIX
+
+Default: `fiotest`
+
+### FIOTEST_CLEANUP
+
+Default: `false`
+This parameter for this test must be `false`. FIO I/O test consist of two steps. In first step fio pods will be created
+and then in second step fio load will be executed against fio pods, so it is important that clusterloader which is used
+to create pods does not delete pods after fist part of test is finished. 
+
+### FIOTEST_BASENAME
+
+Default: `fiotest`  
+Basename used by cluster loader for the project(s) it creates.
+
+### FIOTEST_MAXPODS
+
+Default: `1`  
+Maximum number of Pods that FIO I/O test will create for test
+
+### FIOTEST_POD_IMAGE
+
+Default: `quay.io/openshift-scale/scale-ci-fio:latest`
+Container image to use for FIO Pods 
+
+### FIOTEST_STEPSIZE
+
+Default: `1`  
+Number of Pods for cluster loader will create before waiting for Pods to become running.
+
+### FIOTEST_PAUSE
+
+Default: `0`  
+Period of time (in seconds) for cluster loader to pause after creating Pods and waiting for them to be "Running" state.
+When `FIOTEST_PAUSE` is zero, cluster loader will create pods in fastest possible manner. 
+
+### FIOTEST_STORAGE_SIZE
+
+Default: `2Gi`
+`FIOTEST_STORAGE_SIZE` defines size of PVC which will be created and mounted to Pod. It is important to notice that this
+cannot be smaller than `FIOTEST_FILESIZE`
+
+### FIOTEST_STORAGECLASS
+
+Default: `gp2`
```

Comment:
The default here is actually set to nothing - https://github.com/openshift-scale/workloads/pull/74/files#diff-9cb48f1d10b76b82da4881a9db01f896R39

We might want to include common defaults for each cloud environment this would be run against. Examples:
AWS - gp2
Azure - managed-premium
OSP - standard

Reply:
added